muttdown/main.py

```diff
@@ -160,10 +166,10 @@ def main():
     if args.print_message:
         print(rebuilt.as_string())
     elif args.sendmail_passthru:
-        cmd = c.sendmail.split() + ['-f', args.envelope_from] + args.addresses
+        cmd = c.sendmail.split() + ['-G', '-i', '-f', args.envelope_from] + args.addresses
 
         proc = subprocess.Popen(cmd, stdin=subprocess.PIPE, shell=False)
-        proc.communicate(rebuilt.as_string())
+        proc.communicate(rebuilt.as_string().encode())
```

Comment:
calling `encode` without an argument has pretty confusing behavior on a lot of systems since it tries to guess the default encoding

Reply:
Arguable, I think the default is actually the least confusing possible behavior. I would opt instead for just dropping support of legacy python, which is really the root cause of confusion around strings/bytes. Especially with the additional decode bound method that converts the html from base64 to usual text encoding.